DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/Shared/_DelegateGroupsSideNavMenu.cshtml

```diff
@@ -18,10 +18,10 @@
                      is-current-page="@(Model.CurrentPage == DelegateGroupPage.Delegates)"
                      asp-all-route-data="@routeData" />
 
-  <vc:side-menu-link asp-action="GroupCourses"
-                     asp-controller="DelegateGroups"
+  <vc:side-menu-link asp-action="Index"
```

Comment:
This is an fix unrelated to HEEDLS-514. Will have been introduced when Group Courses was refactored into it's own controller.

Reply:
Yeah, looks like I missed this one when I was refactoring as part of 517, and the ticket hasn't been tested yet